airbyte-integrations/connectors/destination-firebolt/bootstrap.md

```diff
@@ -19,4 +19,6 @@ This connector uses [firebolt-sdk](https://pypi.org/project/firebolt-sdk/), whic
 ## Notes
 
 * Integration testing requires the user to have a running engine. Spinning up an engine can take a while so this ensures a faster iteration on the connector.
-* S3 is generally faster writing strategy and should be preferred.
\ No newline at end of file
+* S3 is generally faster writing strategy and should be preferred.
+* For normalization, quotes (") in json will be escaped \" and appear so in the final table. e.g. _airbyte_raw table contains `{"column": "my\"value\""}` final table would contain `my\"value\"` in column "column", therefore potentially diverging from original data.
+* Array type is curently not supported for normalization due to a limitation on the SQL side.
```

Comment:
> the SQL side

does the `unnest` macro in arrays.sql not cover this?

Reply:
Unfortunately no, as this is more to do with an internal bug on how nullability of a column is detected on the Firebolt side when `json_extract_array` is called. I'll try to clarify this line in the doc.